pandas/core/window/expanding.py

```diff
@@ -209,10 +209,10 @@ def skew(self, **kwargs):
 
     >>> arr = [1, 2, 3, 4, 999]
     >>> import scipy.stats
-    >>> fmt = "{0:.6f}"  # limit the printed precision to 6 digits
-    >>> print(fmt.format(scipy.stats.kurtosis(arr[:-1], bias=False)))
+    >>> precision = 6  # limit the printed precision to 6 digits
```

Comment:
Minor but I find this rather distracting from the code sample. Can you just hard-code the format specifier in the print statements below?

Reply:
Sounds good - just did that.